README.md

```diff
@@ -72,37 +74,41 @@ In any extra time:
     - All functions have docstrings that act as [professional-quality documentation](http://google.github.io/styleguide/pyguide.html#381-docstrings). 
     - If applicable, [well documented](https://www.sqlstyle.guide/) SQL queries with appropriate single-line or multiline comments.
     - Quality linear regression model
-       - Your model should not violate any of the assumptions of a linear regression
+       - Your model should avoid violating any of the assumptions of a linear regression
        - Whenever necessary, briefly explain in comments the changes made from one iteration to the next, and why you made these choices
 4. A standalone `data/` directory that stores all relevant raw and processed data files
-    - Be sure to include how the data was obtained!
+    - **Be sure to include how the data was obtained!**
     - All large files are labeled in the `.gitignore` file to avoid having them accidentally live in your commit history.
 5. A standalone `references/` directory that stores all relevant literature, data dictionaries, or useful references that were used to help you during the project.
     - Use this directory to store physical copies of the `.pdf` files; or
     - Create a `README.md` file that cites external resources that were used.
 6. A standalone `reports/` directory that stores your `memo.md` and `presentation.pdf` files
-7. A user-focused `README.md` file that explains your process, methodology and findings.
-    - Take the time to make sure that you craft your story well, and clearly explain your process and findings in a way that clearly shows both your technical expertise and your ability to communicate your results!
-8. One Jupyter Notebook file that focuses on EDA, visualization, and presentation. 
+7. A standalone `notebooks/` directory that stores both your exploratory and report notebooks
+    - A record of your workflow should be stored in `notebooks/exploratory`. Don't be afraid to leave in error messages, so you know what didn't work!
+8. A user-focused `README.md` file that briefly covers your process, methodology and findings.
+    - Someone with no context on your project should be able to use this document to understand the structure of your project, and adapt your code for their needs.
+9. One final Jupyter Notebook file stored in `notebooks/report` that focuses on visualization and presentation
     - The very beginning of the notebook contains a description of the purpose of the notebook.
        - This is helpful for your future self and anyone of your colleagues that needs to view your notebook. Without this context, you’re implicitly asking your peers to invest a lot of energy to help solve your problem. Help them by enabling them to jump into your project by providing them the purpose of this Jupyter Notebook.
```

Comment:
This last sentence is a little awkward. How about we just delete the 'by enabling them' bit so that it reads: 'Help them to jump into your project ...'?